### PR TITLE
otg ate kne main

### PR DESCRIPTION
**Pull request by biplamal**

### CODE REVIEW

File: tests/bgp_route_install_test.go [hunk 1]
```diff
@@ -96,18 +96,18 @@ func routeInstallConfigureBGP(t *testing.T, dut *ondatra.DUTDevice) {
 func routeInstallConfigureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	t.Logf("Start Setting DUT Config")
 	routeInstallConfigureInterface(t, dut)
-	helpers.ConfigDUTs(map[string]string{"arista1": "../resources/dutconfig/bgp_route_install/set_dut.txt"})
+	helpers.ConfigDUTs(map[string]string{"arista": "../resources/dutconfig/bgp_route_install/set_dut.txt"})
```
Comment (greg-dennis) [inline tests/bgp_route_install_test.go:99]: Can you remove the helpers.ConfigDUTs function and use the Ondatra Config API instead? This should work:
`dut.Config().New().
  WithAristaFile("../resources/dutconfig/bgp_route_install/set_dut.txt").
  Push(t)`

Comment (biplamal) [reply tests/bgp_route_install_test.go]: done

File: tests/helpers/webbtest.go [hunk 2]
```diff
@@ -125,7 +120,7 @@ func BgpDeleteNbr(nbrs []*BgpNeighbor) *oc.NetworkInstance_Protocol_Bgp {
 func VerifyPortsUp(t *testing.T, dev *ondatra.Device) {
 	t.Helper()
 	for _, p := range dev.Ports() {
-		status := dev.Telemetry().Interface(InterfaceMap[p.Name()]).OperStatus().Get(t)
+		status := dev.Telemetry().Interface(p.Name()).OperStatus().Get(t)
```
Comment (SuryyaKrJana) [inline tests/helpers/webbtest.go:123]: Just an query, whether we can use Telemetry() in our Athena-G, now? From this diff I could not trace out the caller of "VerifyPortsUp"

**Review by SuryyaKrJana — APPROVED** [review]

File: tests/helpers/webbtest.go (hunk 2 above)
Comment (biplamal) [reply tests/helpers/webbtest.go:123]: its just, checking DUT interfaces are UP or not, that's it via DUT gNMI

**Pull request merged**